test/e2e-tests.sh

```diff
@@ -35,4 +37,46 @@ header "Running Go e2e tests"
 go_test_e2e ./test || failed=1
 
 (( failed )) && fail_test
+
+go build github.com/tektoncd/cli/cmd/tkn
+
+./tkn pipeline list && fail_test "Failed to list pipelines"
+./tkn pipeline describe foo && fail_test "Failed to describe pipeline"
+#./tkn pipelinerun list && fail_test "Failed to list pipelinerun"
+./tkn pipelinerun describe foo && fail_test "Failed to describe pipelinerun"
+#./tkn pipelinerun logs foo && fail_test "Failed to show logs"
+./tkn task list && fail_test "Failed to list task"
+./tkn taskrun logs foo && fail_test "Failed to show logs"
+#./tkn taskrun list && fail_test "Failed to list taskrun"
+
+install_pipeline_crd
+
+# change default namespace to tektoncd
+kubectl create namespace tektoncd
+kubectl config set-context $(kubectl config current-context) --namespace=tektoncd
+
+# create pipeline, pipelinerun, task, and taskrun
+kubectl apply -f ./test/resources/output-pipelinerun.yaml
+kubectl apply -f ./test/resources/task-volume.yaml
+
+notready=true
+while $notready
+  do
+    sleep 2 
+    pr=$(kubectl get pipelinerun/output-pipeline-run -o json)
+    if [ "$(echo "$pr" | jq -r .status.conditions[0].status)" = "True" ]; then
```

Comment:
Nice!!

Just one more change?
If the `status` is false its better to exit with `non-zero ` and log the right failure message. So in this way, it would also prevent never ending loop in case of failure.

Reply:
Also, it would be better to add similar checks for the TaskRun status?

Reply:
indeed :angel: